Source/RapyutaSimulationPlugins/Public/Robots/RRRobotVehicleROSController.h

```diff
@@ -49,4 +49,10 @@ class RAPYUTASIMULATIONPLUGINS_API ARRRobotVehicleROSController : public AAICont
 
     UFUNCTION(BlueprintCallable)
     void SubscribeToMovementCommandTopic(const FString& InTopicName);
+
+    UPROPERTY(BlueprintReadWrite)
+    bool PublishOdom = true;
+
+    UPROPERTY(BlueprintReadWrite)
+    bool PublishOdomTf = false;

```

Comment:
By convention, we could prefix these members with `b` just for quickly recognizing them as boolean. And if making them as `uint8 <bMember> :1` style also (like in various places in UE sources) due to this class being attached to every single robot, we would then have to initialize them in ctor.., which is kinda a catch.